README.md

```diff
@@ -1,93 +1,91 @@
 
-# Signalen
+# Signalen frontend
 
-This project is based on the react boilerplate that was adapted for the needs of the municipality of Amsterdam.
-See https://github.com/Amsterdam/react-boilerplate.git for more information on the react boilerplate
+This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
 
-## Requirements
+## Setup a development environment
 
-  -  npm >= 6.11
-  -  node 12.18
+Make sure you installed the following versions on your machine:
 
-This project relies on the[ `leaflet-headless`](https://www.npmjs.com/package/leaflet-headless) package. On OSX, you'll need to have [`cairo`](https://formulae.brew.sh/formula/cairo), [`pango`](https://formulae.brew.sh/formula/pango) and [`cmake`](https://formulae.brew.sh/formula/cmake) installed.
+-  npm >= 6.11
+-  node 12.18
 
-## Installation
+Install the dependencies:
 
-  -  npm install
+```bash
+npm install
+```
 
-## Configuration
+And start the watch server:
 
-Configuration for theming, map and API endpoint URLs is defined in
-`environment.conf.json`.
+```bash
+npm start
+```
 
-## Development
+The server listens runs on port 3001 by default. You can change the by setting the environment variable `PORT`.
 
-To overwrite some configuration for development purposes you can define them in
-`environment.conf.development.json`. This file is ignored by git.
+## Configuration
 
-The SPA will run over HTTP on `localhost` at port `3001`. The port can be configured by setting the environment variable `PORT`.
+Configuration for theming, map and API endpoint URLs is defined in `environment.conf.json`. Override the default configuration by creating the file `environment.conf.development.json`. This file is ignored by Git.
```

Comment:
Might be good to mention that the configuration is inject by Webpack and that changes to the configuration file will only be picked up when the dev server is restarted.